### PR TITLE
Hide "delete_selected" action from actions if has_delete_permission is False

### PR DESCRIPTION
**Pull request by itcrab**
Hi, guys!

Today I found some little problem in django admin.

I have model:
```Python
class Question(models.Model):
    question_text = models.CharField(max_length=200)
    pub_date = models.DateTimeField('date published')
```
And I have admin-class:
```Python
class QuestionAdmin(admin.ModelAdmin):
    def has_delete_permission(self, request, obj=None):
        return False
```

If I open any object `Question` I don't see button "Delete".

But if I open list with objects `Question` in: http://localhost:8000/admin/polls/question/
I see group actions select with item "Delete selected questions"...

If I select some `Question` objects, select item "Delete selected questions" in group actions and click "Go" I see page with this message:
`403 Forbidden`

I think this is not good - why we give opportunity for select group action if we don't have permissions for this action?!

This PR about this.

### CODE REVIEW

**Comment by timgraham** [comment]
Thanks for the contribution. All fixes like this need a Trac ticket. The good news is that one already exists:
https://code.djangoproject.com/ticket/10609
and one that's marked as a duplicate of that one:
https://code.djangoproject.com/ticket/13709
and a related issue:
https://code.djangoproject.com/ticket/11383

Hope that helps gives some background. It would be nice to try for a more general solution that solves the first ticket. Anyway, I'm going to close this PR because all pull requests should go to the master branch rather than stable/1.11.x. This change doesn't qualify for a backport based on our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions).

**Comment by itcrab** [comment]
@timgraham close PR is fantastic!

I right understand that this PR need be:
- paste Trac ticket number;
- from master branch.

Right?

**Comment by timgraham** [comment]
Yes, and as I said, a more general solution that could be applied to other admin actions (as discussed in the ticket) would be better than the solution you've proposed here. 

See also our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).

**Comment by itcrab** [comment]
I gave working solution for fix specific problem with tests for Django 1.11. It's works right now.

You know better - waited 8-7 years (open tickets) and will wait yet.

**Comment by itcrab** [comment]
@timgraham I right understand than if I paste ticket number and branch from master you don't see any opportunities for merge it in Django?

File: django/contrib/admin/options.py [hunk 1]
```diff
@@ -800,6 +800,12 @@ def get_actions(self, request):
             for func, name, desc in actions
         )
 
+        # Hide actions without permissions
+        model_perms = self.get_model_perms(request)
+        if not model_perms['delete']:
```
Comment (timgraham) [inline django/contrib/admin/options.py]: I don't see the advantage of going through `get_model_perms()`. How about: 
`if 'delete_selected' in actions and not self.has_delete_permission(request):`

**Pull request closed without merging**